lib/metadata.ts

```diff
@@ -1,10 +1,15 @@
 import { FetchAppDetailsDocument, FetchAppDetailsQuery } from "../generated/graphql";
-import { getEnvVars } from "./environment";
 import { createClient } from "./graphql";
+import { apl } from "./saleorApp";
 
 export const getValue = async (saleorDomain: string, key: string) => {
+  const authData = await apl.get(saleorDomain);
+  if (!authData) {
+    throw Error("Unknown domain.");
```

Comment:
Not sure about this error message. Domain *was* known and it was provided in argument. Maybe error should exactly say that authData for this domain was not found?